WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -30,6 +31,14 @@
 NSString * const kWPNewPostURLParamTagsKey = @"tags";
 NSString * const kWPNewPostURLParamImageKey = @"image";
 
+// Constants for the unread notification dot icon
+static NSInteger const kNotificationBadgeIconSize = 10;
+static NSInteger const kNotificationBadgeIconVerticalOffsetFromTop = 5;
+static NSInteger const kNotificationBadgeIconHorizontalOffsetFromCenter = 8;
+static NSInteger const kNotificationBadgeIconHorizontalOffsetForIPadInPortrait = 104;
+static NSInteger const kNotificationBadgeIconHorizontalOffsetForIPadInLandscape = 232;
+static NSInteger const kNotificationBadgeIconHorizontalOffsetForIPhone6PlusInLandscape = 93;
```

Comment:
In this very same file, we've got three prefixes for constants: `WP*`, `kWP*`, and just `k*`.

Can we use just one kind of prefix perhaps?. Just checked with @diegoreymendez and he mentioned that constants shouldn't have a `k`. 

How about... `WP*` ?

(Thanks!)